Documentation/gettingstarted/flannel-integration.rst

```diff
@@ -0,0 +1,127 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _flannel-integration:
+
+**************************************
+Cilium integration with Flannel (beta)
+**************************************
+
+This guide contains the necessary steps to run Cilium on top of your Flannel
+cluster.
+
+If you have a cluster already set up with Flannel you will not need to install
+Flannel again.
+
+This Cilium integration with Flannel was performed with Flannel 0.10.0 and
+Kubernetes >= 1.9. If you find any issues with previous Flannel versions please
+feel free to reach us out to help you.
+
+Flannel installation
+--------------------
+
+NOTE: If ``kubeadm`` is used, then pass ``--pod-network-cidr=10.244.0.0/16`` to
+``kubeadm init`` to ensure that the ``podCIDR`` is set.
+
+.. parsed-literal::
+
+  kubectl -f  \ |SCM_WEB|\/examples/kubernetes/addons/flannel/flannel.yaml
+
+
+Wait until all pods to be in ready state before preceding to the next step.
+
+Cilium installation
+-------------------
+
+Download Cilium kubernetes descriptor for your Kubernetes version.
+
+
+.. tabs::
+  .. group-tab:: K8s 1.13
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.13/cilium.yaml
+
+  .. group-tab:: K8s 1.12
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.12/cilium.yaml
+
+  .. group-tab:: K8s 1.11
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.11/cilium.yaml
+
+  .. group-tab:: K8s 1.10
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.10/cilium.yaml
+
+  .. group-tab:: K8s 1.9
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.9/cilium.yaml
+
+  .. group-tab:: K8s 1.8
+
+    .. parsed-literal::
+
+      curl -LO \ |SCM_WEB|\/examples/kubernetes/1.8/cilium.yaml
+
+Edit the ConfigMap in that file and set the option ``flannel-master-device`` with ``"cni0"``.
+
+Set ``flannel-uninstall-on-exit`` with either ``true`` or ``false``. If you
+plan to deploy Cilium and ensure the policy enforcement will always be kept even
```

Comment:
Phrasing: "ensure that policy enforcement will persist even if you remove Cilium"